### DISCUSSION TITLE
Tedious flashcard tagging

### DISCUSSION THREAD

**Discussion by Truestarsense** [st3v3nmw/obsidian-spaced-repetition#882] [General]
I'm finding it really tedious to have to write a tag in every note that I want to mark as a deck. Adding a command so that I don't have to go digging into my settings and name a new deck tag every time I want to set up a new deck would make this plugin a lot more usable. Better yet simply get rid of the tag system all together and have the plugin search for flashcard separators globally.

**Comment by ronzulu** [comment]
Hi @starlogic 

Turning on the following option means that you don't need to tag every note.

![image](https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/75528127/7d7508cd-f0a6-46bc-9fdb-fa7621d70f0e)
